man/coredumpctl.xml

```diff
@@ -119,6 +119,14 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-d</option> <replaceable>DBG</replaceable></term>
+        <term><option>--debugger=</option><replaceable>DBG</replaceable></term>.
```

Comment:
not a fan of needless abbreviations, hence please just use `DEBUGGER` rather than `DBG`